sktime/forecasting/reservoir.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements Next generation reservoir computing estimator."""
+
+__all__ = ["NextRC"]
+__author__ = ["frthjf"]
+
+import numpy as np
+import sklearn.linear_model
+from sklearn.utils.validation import check_array
+
+from sktime.base import BaseEstimator
```

Comment:
Does `BaseEstimator` follow `BaseForcaster` extending forecasting template? Silly question.

Reply:
you'll need to inherit from `BaseForecaster` and follow the `forecasting_simple` extension template. That also explains the tags.

Reply:
Wholesome. Thanks.